### DISCUSSION TITLE
Tickets should transfer from the reporter to who transferred the ticket to them

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#1256] [Ideas]
If I'm working on someone else's ticket and transfer it to the reporter for more info, their only option is to transfer it back to the author. It should transfer to the user who initially transferred it to them. This way we get an actual reliable notification.

Maybe it would be easier to allow devs the option to "claim" a ticket if it meets certain criteria (inactive dev, ticket is 2+ months old, etc)? This would also be helpful for Devember, DevQuests, etc. where we have an unofficial dibs system.

**Comment by redwizard42** [comment]
It occurs to me that I transferred a ticket to the user on a ticket I was looking at for Devember and then.. I don't remember what ticket it was. But what if they responded and transferred it back and I'll never know?
